texturepacker/pluginsrc/com/defold/bob/pipeline/tp/AtlasBuilder.java

```diff
@@ -201,33 +215,47 @@ static private TextureSetLayout.SourceImage createSprite(Info.Sprite srcSprite)
         {
             width = originalSize.getHeight();
             height = originalSize.getWidth();
+
+            // When rotated, the "top left" of the image is now the top right
+            // and we need to use the tight rect to find it
+            float corner_x = tightRect.getX() + tightRect.getWidth() + offsety;
+            float corner_y = tightRect.getY() - offsetx;
+            x = corner_x - originalSize.getHeight();
+            y = corner_y;
```

Comment:
Reconstructing the rectangle when it's rotated, and also uses an offset.
Perhaps that's something that the .tpinfo format should provide, but that's for a future task.

Reply:
should we have a separate ticket for that to keep it in mind?

Reply:
Added https://github.com/defold/extension-texturepacker/issues/7